Support FUSE-T

[`FUSE-T`](https://github.com/macos-fuse-t/fuse-t) is supposed to be a drop-in replacement for [`macfuse`](https://osxfuse.github.io/). It is a kext-less implementation of FUSE, making it easier to install on Mac.

Please consider adding support for it. Right now, when running `hubfs`, it panics `cgofuse: cannot find FUSE`.

It looks like `winfsp/cgofuse` already [checks for `libfuse-t`](https://github.com/winfsp/cgofuse/blob/b8358bce7bce407a8f5a9445723df52ebca32337/fuse/host_cgo.go#L178), but the panic comes from "github.com/billziss-gh/cgofuse/" instead.

Full stack:

```
hubfs -o uid=-1,gid=-1,default_permissions,noapplexattr github.com Projects
panic: cgofuse: cannot find FUSE

goroutine 1 [running]:
github.com/billziss-gh/cgofuse/fuse.(*FileSystemHost).Mount(0xc0002fc200, {0x20894c756, 0x0}, {0xc0002fc180, 0x200000101, 0x4})
        /Users/runner/go/pkg/mod/github.com/billziss-gh/cgofuse@v1.5.0/fuse/host.go:610 +0x7f8
main.mount({0x4403948, 0xc000116000}, {0x0, 0x0}, {0x20894c756, 0x8}, {0xc0002fc140, 0x4, 0x12})
        /Users/runner/work/hubfs/hubfs/src/main.go:115 +0x265
main.run()
        /Users/runner/work/hubfs/hubfs/src/main.go:296 +0x13ca
main.main()
        /Users/runner/work/hubfs/hubfs/src/main.go:305 +0x19
```

--- a/Contributors.asciidoc
+++ b/Contributors.asciidoc
@@ -55,4 +55,5 @@ CONTRIBUTOR LIST
 ----------------
 |===
 |Bill Zissimopoulos                     |billziss at navimatics.com
+|Sơn Trần-Nguyễn                        |github at sntran.com
 |===

--- a/src/go.mod
+++ b/src/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/cli/browser v1.0.0
 	github.com/cli/oauth v0.9.0
 	github.com/go-git/go-git/v5 v5.2.0
-	github.com/winfsp/cgofuse v1.5.1-0.20220421173602-ce7e5a65cac7
+	github.com/winfsp/cgofuse v1.6.0
 )
 
 replace github.com/go-git/go-git/v5 v5.2.0 => github.com/billziss-gh/go-git/v5 v5.2.1-0.20210325075736-c1624bffeb12

